contracts/evoting/types/admin.go

```diff
@@ -73,58 +70,63 @@ func (adminForm *AdminForm) GetAdminIndex(userID string) (int, error) {
 }
 
 // RemoveAdmin add a new admin to the system.
-func (adminForm *AdminForm) RemoveAdmin(userID string) error {
-	index, err := adminForm.GetAdminIndex(userID)
+func (adminList *AdminList) RemoveAdmin(userID string) error {
+	index, err := adminList.GetAdminIndex(userID)
 	if err != nil {
-		return xerrors.Errorf("Failed to retrieve the Admin permission: %v", err)
+		return xerrors.Errorf("Failed GetAdminIndex: %v", err)
 	}
 
 	if index < 0 {
 		return xerrors.Errorf("Error while retrieving the index of the element.")
 	}
 
```

Comment:
please change this to more intuitive error messages (if I don't know the code very well, I don't know what is `GetAdminIndex` or what it means that the index could not be found)

Reply:
also changed back